src/app/pages/start-wizard/start-wizard.page.html

```diff
@@ -1,265 +1,136 @@
-          </ion-col>
-        </ion-row>
-        <ion-row class="legal-text-row">
-          <ion-col align-self-top>
-            <h2 [@headerAnimation5]="state">{{'LEGAL_TERMS.HEADER' | translate}}</h2>
-            <h3 class="ion-no-padding ion-text-wrap">
-              <ul>
-                <li [@legalText1]="state">{{'LEGAL_TERMS.LINE_1' | translate}}</li>
-                <li [@legalText2]="state">{{'LEGAL_TERMS.LINE_2' | translate}}</li>
-                <li [@legalText3]="state">{{'LEGAL_TERMS.LINE_3' | translate}}</li>
-                <li [@legalText4]="state">
-                  <span [innerHtml]="'LEGAL_TERMS.LINE_4' | translate"></span>
-                </li>
-                <li [@legalText5]="state">
-                  <span [innerHtml]="'LEGAL_TERMS.LINE_5' | translate"></span>
-                </li>
-                <li [@legalText6]="state">
-                  <span [innerHtml]="'LEGAL_TERMS.LINE_6' | translate:{legalUrl: legalUrl}"></span>
-                </li>
-              </ul>
-            </h3>
-          </ion-col>
-        </ion-row>
-      </ion-grid>
-    </ion-slide>
-  </ion-slides> -->
-</ion-content>
+<swiper-container
+  #swiper
+  slides-per-view="auto"
+  loop="false"
+  css-mode="false"
+  navigation="true"
+  pagination="true"
+  space-between="0"
+  centered-slides="false"
+  center-insufficient-slides="false"
+  zoom="true"
+  keyboard="false"
+  (swiperslidechange)="onSlideChange($event)"
+>
+  <swiper-slide>
+    <div class="slide__container">
+      <div class="svg__container">
+        <svg-icon src="/assets/images/onboarding/eclipse.svg"></svg-icon>
+      </div>
+      <div class="text__container" style="max-width: 230px">
+        <h2>
+          {{ "SETTINGS.LANGUAGE" | translate }}
+          @if (language !== LangKey.en) {
+            <span> / Language</span>
+          }
+        </h2>
+        <ion-list class="language-list">
+          <ion-item class="ion-no-padding">
+            <ion-label position="stacked" color="medium">
+              {{ "START_WIZARD.CHOOSE_LANGUAGE.HEADER" | translate }}
+              @if (language !== LangKey.en) {
+                <span> / Choose language</span>
+              }
+            </ion-label>
+            <ion-select
+              cancelText="{{ 'DIALOGS.CANCEL' | translate }}"
+              [(ngModel)]="language"
+              [interface]="!isDesktop ? 'action-sheet' : 'popover'"
+              (ngModelChange)="saveLanguage()"
+            >
+              @for (lang of supportedLanguages; track lang.langKey) {
+                <ion-select-option [value]="lang.langKey">
+                  {{ lang.name }}
+                </ion-select-option>
+              }
+            </ion-select>
+          </ion-item>
+        </ion-list>
+      </div>
+    </div>
+  </swiper-slide>
+  <swiper-slide>
+    <div class="slide__container">
+      <div class="svg__container">
+        <svg-icon src="/assets/images/onboarding/line.svg"> </svg-icon>
```

Comment:
Er det ikke bedre å vise svg-en på en annen måte? svg-icon er jo tenkt brukt til ikoner, så dette er lett å misforstå.
Ett alternativ er jo bare å inline svg-en, hvis ikke filene er veldig store (kjør de evt gjennom svgomg først hvis ikke du har gjort det alt..).

Reply:
ok, jeg skal bruke vanlig img. tror de svg filene er ganske store for å bruke de direkte